src/interfaces/IComet.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: BUSL 1.1
+pragma solidity 0.8.13;
+
+/// @dev Taken from https://github.com/compound-developers/compound-3-developer-faq/blob/master/contracts/MyContract.sol
+
+interface IComet {
```

Comment:
Is there a commented version of this interface perchance?

Reply:
unfortunately, there is not. here is the prod version: https://github.com/compound-finance/comet/blob/main/contracts/CometMainInterface.sol